torch/_functorch/partitioners.py

```diff
@@ -1018,44 +798,45 @@ def should_ban_recomputation(node):
             return False
         if node.target == operator.getitem:
             return False
+        if op_types.is_view(node):
+            return False
```

Comment:
This seems new ... just pointing that out ..

Reply:
ah yeah, I guess this is the only semantics change. I changed it so that we never save views for the backwards. I might split that into a separate PR.